build.gradle

```diff
@@ -69,4 +67,32 @@ project(':configurator') {
               'Main-Class': mainClassName)
     }
   }
+}
+
+project(':launcher') {
+
+  mainClassName = 'org.apereo.openequella.adminconsole.launcher.ClientLauncher'
+
+  apply plugin: "application"
+
+  group = 'org.apereo.openequella.adminconsole'
+
+  dependencies {
+    compile project(':common')
```

Comment:
just a question here...as compile is deprecated in gradle's new version, we can use implementation instead. But here compile is still used. Is it because what is dependent here is a subproject?

Reply:
I have no idea :)  I don't know anything about Gradle.  I will look into it.

Reply:
The build script is now a single project and there is no reference to compile